scripts/build.sh

```diff
@@ -94,11 +94,11 @@ function CreateDirectory {
 
 function Generate {
   conan config install https://github.com/novelrt/ConanConfig.git
-  conan install . -if "$BuildDir" --build=missing --build=bison --profile linux-clang10-amd64
+  conan install . -if "$BuildDir" --build=missing --profile linux-clang10-amd64
```

Comment:
This works now?

Reply:
Looks like it.

Reply:
We should update the README to reflect this then, probably.